generate_tuxsuite.py

```diff
@@ -30,14 +30,18 @@ def get_repo_ref(config, tree_name):
 
 
 def emit_tuxsuite_yml(config, tree):
+    tuxsuite_yml = "tuxsuite/{}.tux.yml".format(tree)
+    orig_stdout = sys.stdout
+    f = open(tuxsuite_yml, "w")
```

Comment:
aren't `with` blocks more canonical python?

Reply:
i.e.
```python
with open(tuxsuite_yml, "w") as f:
    <indent rest of code that needs f in scope>
```
then you can drop the explicit close method call.

Reply:
Sure, if that is what you prefer. I did it this way as it is less indenting.